src/amp.js

```diff
@@ -42,6 +43,7 @@ import {maybeTrackImpression} from './impression';
 try {
   // Should happen first.
   installErrorReporting(window);  // Also calls makeBodyVisible on errors.
+  installCacheServiceWorker(window);
```

Comment:
We can call it whenever. Are you suggesting later in the try-catch, or as a `timer.delay`?